physics/src/modules/include/IOceanBoundary.hpp

```diff
@@ -13,12 +13,12 @@
 namespace Nextsim {
 
 namespace CouplingFields {
-constexpr TextTag SST = "SST"; // sea surface temperature ˚C
-constexpr TextTag SSS = "SSS"; // sea surface salinity PSU
-constexpr TextTag MLD = "MLD"; // Mixed layer or slab ocean depth m
-constexpr TextTag OCEAN_U = "U"; // x(east)-ward ocean current m s⁻¹
-constexpr TextTag OCEAN_V = "V"; // y(north)-ward ocean current m s⁻¹
-constexpr TextTag SSH = "SSH"; // sea surface height, m
+    constexpr TextTag SST = "SST"; // sea surface temperature ˚C
+    constexpr TextTag SSS = "SSS"; // sea surface salinity PSU
+    constexpr TextTag MLD = "MLD"; // Mixed layer or slab ocean depth m
+    constexpr TextTag OCEAN_U = "U"; // x(east)-ward ocean current m s⁻¹
+    constexpr TextTag OCEAN_V = "V"; // y(north)-ward ocean current m s⁻¹
+    constexpr TextTag SSH = "SSH"; // sea surface height, m
```

Comment:
If this is `clang-format`, then it is fine, but I thought that a `namespace` block didn't typically gain a level of indentation.

Reply:
They do gain a level of indentation when nested in other namespaces (see [here](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#namespaceindentation))